.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea
+*.iml
+build
```

Comment:
Please add .class and other extensions here. Only source code should be committed/pushed/merged